src/layer/vector/Canvas.js

```diff
@@ -215,7 +215,7 @@ L.Canvas = L.Renderer.extend({
 	},
 
 	_onMouseMove: function (e) {
-		if (!this._map || this._map.dragging._draggable._moving || this._map._animatingZoom) { return; }
+		if (!this._map || (this._map.dragging._enabled && this._map.dragging._draggable._moving) || this._map._animatingZoom) { return; }
```

Comment:
This becomes a bit hard to follow, I wonder if we should not have a method on the L.Draggable object to make this more readable.
Something like `this._map.dragging.moving()` that would do both checks.